lighthouse-core/test/audits/__snapshots__/script-treemap-data-test.js.snap

```diff
@@ -0,0 +1,590 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ScriptTreemapData audit squoosh fixture has root nodes 3`] = `
```

Comment:
I'm still a little dubious that anyone is going to look at this file or understand what changes are bad here. You're sure this isn't a case of anything that's here is valid? :)

Reply:
> You're sure this isn't a case of anything that's here is valid? :)

huh?

Reply:
I pulled out explicit checks for the things that should be static. Otherwise, changes here in the future might be scoped to new properties in a node, or changes in how we determine how big a module is inside a bundle.

Reply:
I guess what I'm asking is, do you foresee any situation in the future where there are changes to this file that should prevent a PR from landing that are not caught by any of our tests?

If the answer is "yes, <blah> could happen" then my next question would be if a small scale explicit test is possible for that. If it's unreasonable then keeping this test sounds great.

If the answer is "no", then this file is mostly here for show, "anything that's here is valid", and we should probably just remove it :)

Reply:
The answer is "I don't know, there could be". It's more of a canary.

Reply:
Gotcha, in that case it feels a bit on the heavy side of churn for a canary we don't know will be helpful, but at least it's documentation of the output for a real source map I suppose. SG